drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java

```diff
@@ -151,8 +152,7 @@ protected String httpDelete(String request) throws IOException {
 	protected String httpPost(String request, JsonObject body) throws IOException, RuntimeException {
 		String host = getKieRESTUrl();
 		URL url = new URL(host + "/" + request);
-		System.out.println("[POST] "+url.toString());
-		System.out.println("[POST] body: "+body);
+		System.out.println("[POST] "+url.toString()+" body: "+body);
```

Comment:
Are System.out statements still actually useful?  Shouldn't they be removed or replaces by some more configurable logging?